SRF/metrics/perf/sierraforest_metrics_perf.json

```diff
@@ -215,5 +243,563 @@
         "MetricGroup": "",
         "MetricName": "llc_miss_remote_memory_bandwidth_write",
         "ScaleUnit": "1MB/s"
+    },
+    {
+        "BriefDescription": "Bandwidth observed by the integrated I/O traffic contoller (IIO) of IO reads that are initiated by end device controllers that are requesting memory from the CPU.",
```

Comment:
`iio_bandwidth_read` introduces a spelling issue `contoller`. We will want to fix it if this description feeds into downstream Linux patches.